src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1427,6 +1432,8 @@ class ApplicationManagerImpl
    */
   resumption::ResumeCtrl resume_ctrl_;
 
+  HmiInterfacesImpl hmi_interfaces_;
```

Comment:
@anosach-luxoft  And this components is not related to request controller 
Only few requests should access this instance and CommandImpl.